### PR TITLE
kms: canonize Go samples

### PR DESCRIPTION
**Pull request by LogicalShark**
Following https://github.com/GoogleCloudPlatform/golang-samples/blob/master/CONTRIBUTING.md

- moved regions into separate files, included imports within regions
- changed logs to io writers, changed return err to return fmt.errorf
- delete functions without region tags not found in codesearch
- add package comment
- add/modify function comments and parameter examples

### CODE REVIEW

**Comment by fhinkel** [comment]
@LogicalShark Can you provide a PR description please. Thanks

File: kms/remove_member_cryptokey.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+	"io"
+
+	"cloud.google.com/go/iam"
+	cloudkms "cloud.google.com/go/kms/apiv1"
+	kmspb "google.golang.org/genproto/googleapis/cloud/kms/v1"
+)
+
+// removeMemberCryptoKeyPolicy removes a specified member from an IAM role for the key
+func removeMemberCryptoKeyPolicy(w io.Writer, name, member string, role iam.RoleName) error {
+	// name: "projects/PROJECT_ID/locations/global/keyRings/RING_ID/cryptoKeys/KEY_ID"
```
[the remark this answers is not in the document]
Comment (LogicalShark) [reply kms/remove_member_cryptokey.go]: Done

File: kms/quickstart/quickstart.go [hunk 2]
```diff
@@ -12,11 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// [START kms_quickstart]
```
Comment (tbpg) [inline kms/quickstart/quickstart.go]: Newline after region tag so it doesn't show up in the GoDoc.

File: kms/decrypt_rsa.go [hunk 3]
```diff
@@ -24,9 +24,12 @@ import (
 	kmspb "google.golang.org/genproto/googleapis/cloud/kms/v1"
 )
 
-// decryptRSA will attempt to decrypt a given ciphertext with an 'RSA_DECRYPT_OAEP_2048_SHA256' private key.stored on Cloud KMS
+// decryptRSA will attempt to decrypt a given ciphertext with an 'RSA_DECRYPT_OAEP_2048_SHA256'
+// private key stored on Cloud KMS.
 func decryptRSA(name string, ciphertext []byte) ([]byte, error) {
 	// name := "projects/PROJECT_ID/locations/global/keyRings/RING_ID/cryptoKeys/KEY_ID/cryptoKeyVersions/1"
+	// cipherBytes, err := encryptRSA(rsaDecryptPath, []byte("Sample message"))
```
Comment (tbpg) [inline kms/decrypt_rsa.go]: It's in a comment, but check this error?

File: kms/quickstart/quickstart.go (hunk 2 above)
Comment (LogicalShark) [reply kms/quickstart/quickstart.go]: Done

File: kms/decrypt_rsa.go (hunk 3 above)
Comment (LogicalShark) [reply kms/decrypt_rsa.go]: Done

**Comment by tbpg** [comment]
Marking as `do not merge` until sample versions are pinned.

File: kms/asymmetric.go [hunk 4]
```diff
@@ -1,281 +0,0 @@
-// Copyright 2019 Google LLC
```
[the remark this answers is not in the document]
Comment (LogicalShark) [reply kms/asymmetric.go]: Sent freeze CL

**Pull request merged**